packages/contentstack-import/package.json

```diff
@@ -9,7 +9,7 @@
     "@contentstack/cli-utilities": "~1.5.10",
     "@contentstack/management": "~1.13.0",
     "@oclif/core": "^2.9.3",
-    "axios": "^1.6.3",
+    "axios": "^1.6.4",
```

Comment:
Can we remove axios from dependency as it's not used in the code?

Reply:
yes

Reply:
done